src/Service.Tests/dab-config.PostgreSql.json

```diff
@@ -34,3485 +34,3485 @@
-                "include": [
-                  "*"
-                ]
-              },
-              "policy": {
-                "request": null,
-                "database": null
-              }
-            },
-            {
-              "action": "create",
-              "fields": null,
-              "policy": {
-                "request": null,
-                "database": null
-              }
-            },
-            {
-              "action": "delete",
-              "fields": null,
-              "policy": {
-                "request": null,
-                "database": null
-              }
-            }
-          ]
-        },
-        {
-          "role": "policy_tester_06",
-          "actions": [
-            {
-              "action": "read",
-              "fields": {
-                "exclude": [],
-                "include": [
-                  "*"
-                ]
-              },
-              "policy": {
-                "request": null,
-                "database": "@item.id eq 1940"
-              }
-            },
-            {
-              "action": "update",
-              "fields": {
-                "exclude": [],
-                "include": [
-                  "*"
-                ]
-              },
-              "policy": {
-                "request": null,
-                "database": null
-              }
-            },
-            {
-              "action": "create",
-              "fields": null,
-              "policy": {
-                "request": null,
-                "database": null
-              }
-            },
-            {
-              "action": "delete",
-              "fields": null,
-              "policy": {
-                "request": null,
-                "database": null
-              }
-            }
-          ]
-        },
-        {
-          "role": "database_policy_tester",
-          "actions": [
-            {
-              "action": "update",
-              "fields": null,
-              "policy": {
-                "request": null,
-                "database": "@item.id ne 1234"
-              }
-            },
-            {
-              "action": "create",
-              "fields": null,
-              "policy": {
-                "request": null,
-                "database": null
-              }
-            },
-            {
-              "action": "read",
-              "fields": null,
-              "policy": {
-                "request": null,
-                "database": "@item.id ne 1234 or @item.id gt 1940"
-              }
-            }
-          ]
-        }
-      ],
-      "mappings": null,
-      "relationships": {
-        "books": {
-          "cardinality": "many",
-          "target.entity": "Book",
-          "source.fields": [],
-          "target.fields": [],
-          "linking.object": null,
-          "linking.source.fields": [],
-          "linking.target.fields": []
-        }
-      }
+  "source": {
```

Comment:
incorrect indentation

Reply:
probably no change is required in this file. maybe just revert the identation change

Reply:
Because of how the config generation happens indentation is problematic, and the indentation on the right is actually "correct"

Reply:
There are some minor stylistic changes and it seems very dependent on which formatter you use 😅